woocommerce/woocommerce-android
# Issue/2295 shipping label info screen

Fixes #2295 by adding two info screens to the print shipping label screens.

#### Screenshots
##### Shipping label format options
<img src="https://user-images.githubusercontent.com/22608780/94549712-427ccd00-0270-11eb-8154-84098bf5010e.png" width="300"/>

##### Print shipping label information screen
<img src="https://user-images.githubusercontent.com/22608780/94549723-44df2700-0270-11eb-8dae-8d63acbc77e6.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/95072486-e232d300-0728-11eb-88e7-86397bde6241.png" width="300"/>

#### To test
- Click on an order from the Orders tab that has shipping labels.
- Click on the `Reprint shipping label` button.
- Click on the `See label layout and paper size options` link and notice the new screen is displayed.
- Click on the `Don't know how to print from your phone?` link and notice the new screen is displayed.

#### Notes
This PR is in draft [this print shipping label PR](https://github.com/woocommerce/woocommerce-android/pull/2909) is reviewed.

Update release notes:
